### ISSUE TITLE
Perfstat parsing error on Raspberry Pi 5

### ISSUE THREAD

**Issue by seppewyns** [open-s4c/benchkit#14]
The function [`_get_available_events`](https://github.com/open-s4c/benchkit/blob/1e0d8a022fb6fc38494710a5c45e0d384d67278b/benchkit/commandwrappers/perf.py#L111-L200) throws an exception when benchkit is run on a Raspberry Pi 5.
This function is called when you pass a custom `events` list to `PerfStatWrap`.

Steps to reproduce:
1. Run the following script:
```python
from benchkit.commandwrappers.perf import PerfStatWrap
perstat_wrapper = PerfStatWrap(events=['cache-misses'])
```

Error trace:
```
Traceback (most recent call last):
  File "/home/pi/performance/wpo5/test.py", line 3, in <module>
    perstat_wrapper = PerfStatWrap(events=['cache-misses'])
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pi/performance/benchkit/benchkit/commandwrappers/perf.py", line 275, in __init__
    events = _validate_events(
             ^^^^^^^^^^^^^^^^^
  File "/home/pi/performance/benchkit/benchkit/commandwrappers/perf.py", line 208, in _validate_events
    all_available_events, _ = _get_available_events(perf_bin=perf_bin)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pi/performance/benchkit/benchkit/commandwrappers/perf.py", line 188, in _get_available_events
    events_dict[current_group][event_id] += " " + event_desc
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
KeyError: 'system_time'
```

This is probably due to an error in the parsing logic of the output from `perf list`.

**Comment by apaolillo** [comment]
Thanks, this should be fixed by #16

**Comment by apaolillo** [comment]
Fixed a while ago by #16

**Comment by apaolillo** [comment]
Fixed a while ago by #16
